### PR TITLE
Intro to limits

### PR DESCRIPTION
**Pull request by MariliaMJ**
Definition, properties and valuable links.

### CODE REVIEW

**Comment by fvcproductions** [comment]
Hey, @MariliaMJ! 

So I'd love to be able to merge your changes but it looks there is an error with the Travis CI build. ⚠️ 

In this case, it looks like you haven't added the right syntax for the title to the article you're creating.

You can reference the [Article Style Guide](https://github.com/freeCodeCamp/guides#article-title) on how to do this correctly.

Once you resolve these issues, I will be able to review your PR and merge it. 😊

File: src/pages/mathematics/limits-intro/index.md [hunk 1]
```diff
@@ -1,15 +1,60 @@
----
-title: Limits Intro
----
+
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: please do not remove the front matter

File: src/pages/mathematics/limits-intro/index.md [hunk 2]
```diff
@@ -1,15 +1,60 @@
...
+Let f(x) = x. Then the limit of f(x) as x tends to 1 is equal to 1.
+
+When one says that f(x) is very close to L, but does not "touch" it, means that their distance is very small, likewise, x tends to p, but not equal to p, means x is at a small distance from p. For that, the definition of absolute value is used.
+
+  |f(x) - L| < ε , |x - p|< δ
+  
+The symbols, epsilon and delta, respectively, represent an arbitrarily small number.
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d1/L%C3%ADmite_01.svg/320px-L%C3%ADmite_01.svg.png" width="200" height="200">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 3]
```diff
@@ -1,15 +1,60 @@
...
+
+That is, whenever a point x is within δ units of c, f(x) is within ε units of L.
+
+#### Properties
+
+Consider that the limits of the functions exist, then:
+
+* **Sum**
+
+<img src="http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties_files/eq0004MP.gif">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 4]
```diff
@@ -1,15 +1,60 @@
...
+
+The limit of a sum is the sum of the limits.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+* **Product**
+
+<img src="http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties_files/eq0005MP.gif">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 5]
```diff
@@ -1,15 +1,60 @@
...
+
+The limit of a product is the product of the limits.
+
+* **Product by a constant**
+
+<img src="http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties_files/eq0003MP.gif">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 6]
```diff
@@ -1,15 +1,60 @@
...
+
+You can “factor” a multiplicative constant out of a limit.
+
+* **Division**
+
+<img src="http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties_files/eq0006MP.gif">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 7]
```diff
@@ -1,15 +1,60 @@
...
+
+The limit of a division is the division of the limits.
+
+* **Power**
+
+<img src="http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties_files/eq0007MP.gif">
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: alt tag

File: src/pages/mathematics/limits-intro/index.md [hunk 8]
```diff
@@ -1,15 +1,60 @@
...
+
+If n is an integer.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+![Notes and further examples](http://tutorial.math.lamar.edu/Classes/CalcI/LimitsProperties.aspx)
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: please remove the `!`

File: src/pages/mathematics/limits-intro/index.md [hunk 9]
```diff
@@ -1,15 +1,60 @@
...
+
+![Intro To limits lecture](https://www.khanacademy.org/math/ap-calculus-ab/ab-limits-continuity/ab-limits-graphically/v/introduction-to-limits-hd)
```
Comment (Bouncey) [inline src/pages/mathematics/limits-intro/index.md]: please remove the `!`

**Comment by 1andee** [comment]
I've reviewed the latest commit and believe this PR now complies with the submission standards, with the exception of the image alt tags previously requested. Thoughts/comments @Bouncey?

**Comment by Bouncey** [comment]
Thanks for the article @MariliaMJ and sorry for the delay

**Pull request merged**